src/Console/Command/Local.php

```diff
@@ -0,0 +1,115 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Console\Command;
+
+use Bref\Bref;
+use Bref\Context\Context;
+use Bref\Runtime\Invoker;
+use Exception;
+use JsonException;
+use Psr\Container\NotFoundExceptionInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Yaml\Yaml;
+use Throwable;
+
+/**
+ * Local function invocation.
+ */
+class Local
+{
+    public const SIGNATURE = 'local [function] [data] [--file=] [--handler=]';
+
+    public function __invoke(?string $function, ?string $data, ?string $file, ?string $handler, SymfonyStyle $io): int
+    {
+        if ($function && $data && $handler) {
+            throw new Exception('You cannot provide both a funtion name and the --handler= option.');
+        }
+
+        if ($handler) {
+            // Shift the arguments since there is no function passed
+            $data = $function;
+        } else {
+            $handler = $this->handlerFromServerlessYml($function);
+        }
+
+        if ($data && $file) {
+            throw new Exception('You cannot provide both event data and the --file= option.');
+        }
+
+        try {
+            $handler = Bref::getContainer()->get($handler);
+        } catch (NotFoundExceptionInterface $e) {
+            throw new Exception($e->getMessage() . PHP_EOL . 'Reminder: `bref local` can invoke functions that use the FUNCTION runtime, not the HTTP (or "FPM") runtime. If you are unsure, check out https://bref.sh/docs/local-development.html#http-applications to run HTTP applications locally.');
+        }
+
+        if ($file) {
+            if (! file_exists($file)) {
+                throw new Exception("The file '$file' does not exist.");
+            }
+            $data = file_get_contents($file);
+        }
+
+        try {
+            $event = $data ? json_decode($data, true, 512, JSON_THROW_ON_ERROR) : null;
+        } catch (JsonException $e) {
+            throw new Exception('The JSON provided for the event data is invalid JSON.');
+        }
+
+        // Same configuration as the Bref runtime on Lambda
+        ini_set('display_errors', '1');
+        error_reporting(E_ALL);
+
+        $requestId = '8f507cfc-example-4697-b07a-ac58fc914c95';
+        $startTime = $this->logStart($io, $requestId);
+
+        try {
+            $invoker = new Invoker;
+            $result = $invoker->invoke($handler, $event, new Context($requestId, 0, '', ''));
+        } catch (Throwable $e) {
+            $io->error($e->getMessage());
+            return 1;
+        }
+
+        $this->logEnd($startTime, $io, $requestId);
+        // Show the invocation result
+        $io->block(json_encode($result, JSON_THROW_ON_ERROR | JSON_PRETTY_PRINT), null, 'fg=black;bg=green', '', true);
+
+        return 0;
+    }
+
+    private function handlerFromServerlessYml(string $function): string
+    {
+        if (! file_exists('serverless.yml')) {
```

Comment:
Do you think is it possible to set the path where is the serverless.yml as a option of the command? For some reasons our serverless.yml is not in the root folder but in a subfolder and if we can not set the folder we should copy the serverless file in the root folder that is not cool for us.

Reply:
Sure that makes sense, we can mirror the `--config` option of `serverless` (`bref local --config=foo/serverless.yml ...`), feel free to send a PR on branch `v1.0`.